app/src/main/java/cm/aptoide/pt/home/apps/list/AppsController.kt

```diff
@@ -88,12 +88,16 @@ class AppsController(val themeManager: ThemeManager) :
 
   private fun getPromotionValue(migrations: List<AppcUpdateApp>): Float {
     var promotionValue = 0f
+    var shouldShow = true
     for (migration in migrations) {
       if (migration.hasPromotion) {
         promotionValue += migration.appcReward
+      } else {
+        shouldShow = false
+        break
       }
     }
-    return promotionValue
+    return if (shouldShow) promotionValue else 0f
```

Comment:
I believe this changes are breaking the UI.

flow:
- install old cats version from external source (ask me)
- go to Apps

Reply:
I believe it is not!!!!!!!!!!!! 🗡️ 

But seriously, I can't reproduce it on my phone